### PR TITLE
[JENKINS-61453] Jenkins.MANAGE user to access "Prepare for Shutdown"

### PR DESCRIPTION
**Pull request by escoem**
See [JENKINS-61453](https://issues.jenkins-ci.org/browse/JENKINS-61453).

### Proposed changelog entries

JENKINS-61453 _Prepare for Shutdown_ management link is now accessible to users with
Jenkins.MANAGE (as well as the usual
Jenkins.ADMINISTRATOR)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] JIRA issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@daniel-beck @oleg-nenashev 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Review by amuniz — APPROVED** [review]
Looks good.

**Review by mikecirioli — APPROVED** [review]
👍

File: core/src/main/java/jenkins/model/Jenkins.java [hunk 1]
```diff
@@ -3915,7 +3915,7 @@ public HttpRedirect doQuietDown(@QueryParameter boolean block, @QueryParameter i
      */
     @RequirePOST // TODO the cancel link needs to be updated accordingly
```
Comment (daniel-beck) [inline core/src/main/java/jenkins/model/Jenkins.java]:
```suggestion
    @RequirePOST
```

Almost certainly obsolete, or we'd know about it.

**Review by daniel-beck — APPROVED** [review]

**Review by fcojfernandez — APPROVED** [review]

**Review by varyvol — APPROVED** [review]

**Review by EstherAF — APPROVED** [review]

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Build queue contains a "cancel" link while Jenkins is quieting down, that link is not displayed to Manage users:

https://github.com/jenkinsci/jenkins/blob/450d87491df16598409d26511bcad02978eaa8bb/core/src/main/resources/lib/hudson/queue.jelly#L54

**Comment by amuniz** [comment]
Looks like requested changes are fixed. @daniel-beck can you please re-review?

**Review by Wadeck — APPROVED** [review]
Not tested manually but seems fine :)

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**